.env

```
@@ -60,6 +60,6 @@ KEYCLOAK_VERSION=26.2
 # Run `node supporting_scripts/sync-client-versions.mjs` after changing these.
 # Note: ANGULAR_VERSION does not update all Angular-related packages; Angular CDK/Material, CLI,
 # and build tooling are excluded from automatic syncing and may need separate updates.
-ANGULAR_VERSION=21.2.9
+ANGULAR_VERSION=21.2.11
 ANGULAR_ESLINT_VERSION=21.3.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANGULAR_ESLINT_VERSION key should go before the ANGULAR_VERSION key

(UnorderedKey)